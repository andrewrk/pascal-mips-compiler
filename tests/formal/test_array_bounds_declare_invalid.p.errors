ERROR: line 3: main class "testArrayBoundsDeclareInvalid" must have a parameterless constructor
ERROR: line 8: invalid array range: [7..2]
