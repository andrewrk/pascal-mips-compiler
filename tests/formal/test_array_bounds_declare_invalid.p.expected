ERROR: line 8: invalid array range: [7..2]
